### PR TITLE
samples: hello arduino: Clean up

### PR DESCRIPTION
**Pull request by DhruvaG2000**
Cleanup hello_arduino sample by removing comments that no longer apply and also removing unnecessary amount of prints.

Closes https://github.com/zephyrproject-rtos/gsoc-2022-arduino-core/issues/38

Signed-off-by: Dhruva Gole <goledhruva@gmail.com>

### CODE REVIEW

File: samples/hello_arduino/src/app.cpp [hunk 1]
```diff
@@ -3,27 +3,13 @@
...
-  ret1 = Serial.print("My letter is: ");
-  ret2 = Serial.println(c);
-  printk("Sizes: %d %d\n", ret1, ret2);
-  Serial.println();
-  char myString[] = "Will it print?";
-  ret1 = Serial.println(myString);
-  printk("Size: %d \n\n\n", ret1);
-  delay(1000); // 1 second delay
+	// loop function for continuously executing code
+	Serial.println("Hello World!");
```
Comment (soburi) [inline samples/hello_arduino/src/app.cpp:13]: It is good as "hello world" example.

The original was also work as formatting test.
Is not need to consider?
(such like https://www.arduino.cc/reference/en/language/functions/communication/serial/print/)

**Review by soburi — COMMENTED** [review]
It is good as "hello world" example.
The original was also work as formatting test.
Is not need to consider?
(Such like as https://www.arduino.cc/reference/en/language/functions/communication/serial/print/.)

**Comment by DhruvaG2000** [comment]
> It is good as "hello world" example. The original was also work as formatting test. Is not need to consider? (Such like as https://www.arduino.cc/reference/en/language/functions/communication/serial/print/.)

I agree that we should have a way to demo other Serial.print variants as well.
Let me then rather than delete all other lines make them look neater.

**Comment by DhruvaG2000** [comment]
Closing this- because this sample actually demonstrates a lot of useful stuff that our API can do so let's keep as is.

**Pull request closed without merging**